src/app/components/navbar/NavBar.jsx

```diff
@@ -69,10 +69,11 @@ const NavBar = props => {
     const changeLang = () => {
         if (previewLanguage == "en") {
             setPreviewLanguage("cy");
+            store.dispatch(setNotEnglish());
```

Comment:
So ideally in my head that is going to the store and using the SET_NOT_ENGLISH action found in the languageReducer, is that correct?